automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -390,6 +390,7 @@ def sync_capsule_repos_to_upgrade(capsules):
     from_version = os.environ.get('FROM_VERSION')
     to_version = os.environ.get('TO_VERSION')
     os_ver = os.environ.get('OS')[-1]
+    os.environ['RHEV_CAPSULE_AK'] = 'rhel6_capsule_ak'
```

Comment:
Doing this will make RHEV_CAPSULE_AK always be `rhel6_capsule_ak`. That said change line 395 to `'CAPSULE_AK', 'rhel6_capsule_ak')`